odoo/addons/base/rng/common.rng

```diff
@@ -231,6 +231,7 @@
             <rng:optional><rng:attribute name="bold"/></rng:optional>
             <rng:optional><rng:attribute name="avg"/></rng:optional>
             <rng:optional><rng:attribute name="select"/></rng:optional>
+            <rng:optional><rng:attribute name="disable_counters"/></rng:optional>
```

Comment:
check in the first version where disable_counters has been introduced if it works. If it doesn't because of rng validation, this change needs to be done as a FIX, in that version

Reply:
XMO has changed things in field validation six month ago, it is maybe related

Reply:
I am going to investigate

Reply:
it was forgotten in the PR https://github.com/odoo/odoo/commit/dd7022eccd99b5ba9688f63b517d23c405f23797#diff-150872ec366afb816e1fe98612a57634R236

Reply:
should be added in version branch 13.0 --> note added in the pad